build.gradle.kts

```diff
@@ -173,7 +173,7 @@ signing {
 }
 
 nexusPublishing {
-	repositories {
+	this.repositories {
```

Comment:
Why `this`?

Reply:
Oh, right. I was looking at the build file in IntelliJ and it marked this line as a 'problem'.

![image](https://github.com/junit-pioneer/junit-pioneer/assets/4085611/bab06d8c-9128-4be0-a7b8-25e8e5012cb3)

Basically the problem is that we reference `nexusPublishing::repositories` in multiple steps (while having a `repositories` function in top level (?). Adding `this` means that we explicitly call `nexusPublishing::repositories` because in this scope `this` refers to `nexusPublishing`.

I am very new to kotlin, sorry. This also works if you prefer:
```
nexusPublishing.repositories {
    sonatype()
}
```

But all my 'knowledge' comes from reading this: https://kotlinlang.org/docs/this-expressions.html#implicit-this so I might be very off.

Reply:
Anyways, this is not even relevant to the issue and the build seems to work fine so we can change this back here if you prefer.

Reply:
Thx for the explanation, didn't know that. 💡